Directory.Packages.props

```diff
@@ -5,8 +5,8 @@
     <PackageVersion Include="IsExternalInit" Version="1.0.3" />
     <PackageVersion Include="Microsoft.Bcl.HashCode" Version="6.0.0" />
     <PackageVersion Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="9.0.0" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.13.0" />
-    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.13.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="[4.11.0]" />

```

Comment:
What do the brackets mean?

Reply:
It means exact version match, see the [NuGet Package versioning - Version ranges](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges) documentation. This ensures that version 4.11.0 will be resolved if another package depends on `Microsoft.CodeAnalysis.CSharp`.